en/frames_multicopter/holybro_x500v2_pixhawk6c.md

```diff
@@ -0,0 +1,159 @@
+# Holybro X500 V2 + Pixhawk 6C (PX4 Dev Kit)
+
+This topic provides full instructions for building the [Holybro X500 V2 ARF Kit](http://shop.holybro.com/x500-v2-kit_p1288.html), also known as the Holybro PX4 Dev Kit.
+
+![The fully built vehicle with props removed](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/kit_no_props.jpg)
+
+
+## Assembly
+
+### Holders
+
+**Screw**-  Sunk Screw M2.5*6 12pcs
+
+1. Insert the hanger rubber ring gasket in each of their respective hangers.
+   Do not use sharp objects to press the rubbers inside.
+
+   @[youtube](https://youtu.be/4Tid-FCP_aI)
+
+
+1. Take the battery mounting board and screw it with the slide bar clip using the Sunk Screw M2.5*6.
+
+   @[youtube](https://youtu.be/9E-rld6tPWQ)
+
+1. Screw 4 hangers to the Platform Board using Sunk Screw M2.5*6.
+
+   @[youtube](https://youtu.be/4qIBABc9KsY)
+
+1. Take the slide bar and insert 4 hangers to screw to the bottom plate later.
+
+   @[youtube](https://youtu.be/CFx6Ct7FCIc)
+
+1.  Now insert the battery holder and payload holders assembled in step 2 & 3
+
+
+### Power Module
+
+**Screw**- Socket Cap Screw M2.5*6 8pcs | Locknut M3 4pcs |Nylon Standoff M3*5 4pcs | Screw M3*14 4pcs
+
+
+
+1. Take the bottom plate and insert 4 M3*14 screws and fasten the nylon standoffs on the same.
+
+   @[youtube](https://youtu.be/0knU3Q_opEo)
+
+1. Place the Power distribution board and use the locknuts to assemble them. The power module PM02 (for Pixhawk 6C) would power this board
+
+   @[youtube](https://youtu.be/IfsMXTr3Uy4)
+
+1. Use Socket Cap Screws M2.5*6 and screw the bottom plate on the 4 hangers (that we inserted in the 2 bars on the 3rd step of HOLDER assembly)
+
+   @[youtube](https://youtu.be/Qjs6pqarRIY)
+
+### Landing Gear
+
+@[youtube](https://youtu.be/mU4vm4zyjcY)
+
+@[youtube](https://youtu.be/7REaF3YAqLg)
```

Comment:
The images don't match to numbers. Can you apply numbers and indentation so that this works as a "flow of actions"